### DISCUSSION TITLE
AMD Radeon(TM) R7 M360

### DISCUSSION THREAD

**Discussion by HILALOZTEMEL** [mlcommons/GaNDLF#442] [Q&A]
Can I train models using AMD Radeon(TM) R7 M360 GPU? How to do ? , I'm using windows operating system , if I'm not mistaken RocM downloads it to ubuntu. what should I do?

**Comment by sarthakpati** [comment]
You could use Windows Subsystem for Linux: https://docs.microsoft.com/en-us/windows/wsl/install
